cirq-google/cirq_google/engine/engine_processor_test.py

```diff
@@ -855,36 +880,37 @@ def test_run_sweep_params(client):
         assert result.job_finished_time is not None
     assert results == cirq.read_json(json_text=cirq.to_json(results))
 
-    client().create_program.assert_called_once()
-    client().create_job.assert_called_once()
+    client().create_program_async.assert_called_once()
```

Comment:
what's going on here? why aren't these async functions being awaited?

Reply:
Not exactly sure what you're asking. Accessing `client().create_program_async` returns an `AsyncMock` object and this is setting up an assertion that that mock gets called once. The code that makes that call will await the result, but there's no way to assert that here.

Reply:
That makes sense; I no longer remember what I was confused about